client/src/components/ListOfProjects/ListOfProjects.css

```diff
@@ -17,3 +17,8 @@
   background-color: darkgray;
   margin-bottom: 15px;
 }
+
+li:hover{
+  box-shadow: 0 6px 4px -2px #808080;
```

Comment:
We should try to use color names wherever possible. I think that based off the design, a more accurate box-shadow would be:
```css
box-shadow: 0 3px 4px 1px gray;
```
https://projects.invisionapp.com/share/Z9DIWX2XM#/screens/257232552

Reply:
Got it. Thanks